stack.yaml.lock

```diff
@@ -0,0 +1,47 @@
+# This file was autogenerated by Stack.
```

Comment:
Please do not commit this file. This project doesn't really use stack for development, i keep `stack.yaml` for convenience of contributors, but I do not guarantee it's up to date.